src/styles/base/_typography-mixins.scss

```diff
@@ -81,12 +81,12 @@
 
 @mixin text-sm {
 	font-size: .888888889rem;
-	line-height: $base-line-height;
+	line-height: $base-line-height-sm;
 }
 
 @mixin text-xs {
 	font-size: .777777778rem;
-	line-height: $base-line-height;
+	line-height: .714285714 * $base-line-height;
```

Comment:
Only concern here would be if `$base-line-height` were to change from `rem` values to relative values at some point, which is something we might want to consider. Still, this is ok for now.